cns/saas/admin_guide/apoctl/auth.adoc

```diff
@@ -148,6 +148,16 @@ For instance:
 
  apoctl auth google --open-with 'Google Chrome'
 
+
+=== `+pc-token+` subcommand
+
+The `+pc-token+` subcommand allows to retrieve a Microsegmentation token using Prisma
+tokens.
+
+Example:
+
+ apoctl auth pc-token --token "$TOKEN"
```

Comment:
@iansk The examples on this page do not have markup making them code blocks. Is that intentional?